gtsam/hybrid/HybridFactorGraph.cpp

```diff
@@ -35,10 +35,12 @@ void HybridFactorGraph::print(const std::string &str,
   std::string nonlinearStr = str + ": NonlinearFactorGraph";
   std::string discreteStr = str + ": DiscreteFactorGraph";
   std::string dcStr = str + ": DCFactorGraph";
+  std::string gStr = str + ": GaussianGraph";
```

Comment:
const

Reply:
not even sure why we have these intermediate variables. Feel free to move them in place after I merge.